### PR TITLE
Add a build badge

### PR DESCRIPTION
**Pull request by bbrto21**
Signed-off-by: Boram Bae <boram21.bae@samsung.com>

### CODE REVIEW

File: .github/workflows/build_changed_packages.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Build changed packages
+name: Build
```
Comment (swift-kim) [inline .github/workflows/build_changed_packages.yml:1]: Why don't you change the file name as well?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # plugins
 
+![Build](https://github.com/flutter-tizen/plugins/actions/workflows/build_changed_packages.yml/badge.svg)
```
Comment (swift-kim) [inline README.md:3]: Linkify?
```suggestion
[![Build](https://github.com/flutter-tizen/plugins/actions/workflows/build_changed_packages.yml/badge.svg)](https://github.com/flutter-tizen/plugins/actions)
```

File: .github/workflows/build_changed_packages.yml (hunk 1 above)
Comment (bbrto21) [reply .github/workflows/build_changed_packages.yml:1]: Oh~ I will changed the file name!

File: README.md (hunk 2 above)
Comment (swift-kim) [reply README.md]: You need to change `build_changed_packages.yml` if you renamed it.

**Review by swift-kim — APPROVED** [review]

**Review by bwikbs — APPROVED** [review]
:+1:

**Review by HakkyuKim — APPROVED** [review]

**Pull request merged**